definitions/divorce/json/CaseField/CaseField-share-a-case.json

```diff
@@ -0,0 +1,18 @@
+[
+  {
+    "LiveFrom": "20/01/2021",
+    "CaseTypeID": "DIVORCE",
+    "ID": "PetitionerOrganisationPolicy",
+    "Label": "Petitioner Solicitor Policy",
+    "FieldType": "OrganisationPolicy",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "20/01/2021",
+    "CaseTypeID": "DIVORCE",
+    "ID": "RespondentOrganisationPolicy",
+    "Label": "Respondent Solicitor Policy",
+    "FieldType": "OrganisationPolicy",
+    "SecurityClassification": "Public"
```

Comment:
Hi @jacmurphy Forgot to remove this. `RespondentOrganisationPolicy ` Not needed at the moment, i think we would leave it out

Reply:
removed